### PR TITLE
deprecated 'olvine' class (misspelled) --- added olivine class

### PR DESCRIPTION
**Pull request by brandonnodnarb**

### CODE REVIEW

**Comment by brandonnodnarb** [comment]
I followed same method as previous (pyroxene, gully, etc.)   Should be ok, but a once over wouldn't hurt.

File: src/matrMineral.ttl [hunk 1]
```diff
@@ -83,10 +83,20 @@ somamin:NonmetallicMineral rdf:type owl:Class ;
                        rdfs:label "nonmetallic mineral"@en .
 
 
+###  http://sweetontology.net/matrMineral/Olivine
+somamin:Olvine rdf:type owl:Class ;
```
Comment (lewismc) [inline src/matrMineral.ttl:87]: @brandonnodnarb please check typo

**Review by lewismc — CHANGES_REQUESTED** [review]

File: src/matrMineral.ttl (hunk 1 above)
Comment (brandonnodnarb) [reply src/matrMineral.ttl]: Ugh.  c/p fail.
Thanks.

**Comment by brandonnodnarb** [comment]
Ack.  thanks.
done.

**Review by lewismc — DISMISSED** [review]

File: src/matrMineral.ttl [hunk 2]
```diff
@@ -83,10 +83,20 @@ somamin:NonmetallicMineral rdf:type owl:Class ;
...
+
+
 ###  http://sweetontology.net/matrMineral/Olvine
 somamin:Olvine rdf:type owl:Class ;
            rdfs:subClassOf somamin:Mineral ;
-           rdfs:label "olvine"@en .
+           rdfs:label "olvine"@en ;
+           owl:deprecated "true"^^xsd:boolean ;
+           rdfs:seeAlso somamin:Olivine ;
+           dcterms:contributor <https://orcid.org/0000-0002-0337-8610> .
```
Comment (dr-shorthair) [inline src/matrMineral.ttl:99]: Add 
```
dcterms:isReplacedBy  <http://sweetontology.net/matrMineral/Olivine> ;
```

**Review by dr-shorthair — CHANGES_REQUESTED** [review]
Add 
```
dcterms:isReplacedBy  <http://sweetontology.net/matrMineral/Olivine> ;
```

File: src/matrMineral.ttl (hunk 2 above)
Comment (brandonnodnarb) [reply src/matrMineral.ttl:99]: added; also removed `rdfs:seeAlso somamin:Olivine ;`

Comment (brandonnodnarb) [reply src/matrMineral.ttl:100]: added  dcterms:isReplacedBy  somamin:Olivine ;   (instead of full path)

**Review by dr-shorthair — APPROVED** [review]

**Comment by brandonnodnarb** [comment]
Deprecating 'olvine' has been addressed in #224

**Pull request closed without merging**